### PR TITLE
Adding Tailwind CSS back into project

### PR DESCRIPTION
**Pull request by JagrajAulakh**

### CODE REVIEW

File: tailwind.config.js [hunk 1]
```diff
@@ -0,0 +1,16 @@
+module.exports = {
+	content: ["./src/**/*.{js,jsx,ts,tsx}"],
+	theme: {
+		extend: {
+			colors: {
+				"primary-color-400": "#1b1464",
```
Comment (hassanuahmad) [inline tailwind.config.js]: can you make these primary-color-900 then 800 then 700 - same for secondary

File: package.json [hunk 2]
```diff
@@ -10,7 +10,7 @@
     "http-server": "^14.1.0",
     "jwt-decode": "^3.1.2",
     "react": "^17.0.2",
-    "react-bootstrap": "^2.1.1",
+    "react-bootstrap": "^2.1.2",
```
Comment (hassanuahmad) [inline package.json:13]: is this still needed in here 👀

Comment (JagrajAulakh) [reply package.json:13]: Will get rid of it once we get stop using bootstrap in the components

**Comment by zainaraza43** [comment]
good work kings

**Pull request merged**